Create ui-c8y-1020-0-11-update-report-icon-in-navigator-6411-graft-release-cd-6474.md (#2139)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-0-11-update-report-icon-in-navigator-6411-graft-release-cd-6474.md

---
date: ""

> title: Update report icon in navigator

Reviewer: ```suggestion
title: Update report icon in navigator issue fix
```

> title: Update report icon in navigator issue fix

Reviewer 2: ```suggestion
title: Update report icon in navigator correctly
```

> title: Update report icon in navigator (#6411) [GRAFT][release/cd] (#6474)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Update report icon in navigator
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59044
version: 1020.0.11
---

> Update report icon in navigator (#6411) [GRAFT][release/cd] (#6474)

Author: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT platform, reports provide valuable insights and visualizations of data. To improve the user experience and make the reports feature more discoverable, the icon for the reports menu item in the navigator has been updated. The new icon aligns better with the overall design and is more intuitive for users to recognize. This change enhances the usability and visual consistency of the platform, making it easier for users to access and work with reports.
```

> Previously, when report with 'Show in navigator' feature enabled has icon changed, icon was not updated in navigator. Currently, icon is updated in navigator as soon as changes are saved.

Reviewer 2: ```suggestion
Previously, when a report with the **Show in navigator** option enabled had its icon changed, the icon was not updated in the navigator. With this fix, the icon is updated in the navigator as soon as changes are saved.
```

> To improve the user experience and make the reports feature more discoverable, the icon for the reports menu item in the navigator has been updated. The new icon aligns better with the overall design and is more intuitive for users to recognize.

Reviewer: ```suggestion
Previously, when report with 'Show in navigator' feature enabled has icon changed, icon was not updated in navigator. Currently, icon is updated in navigator as soon as changes are saved.
```
